src/definitions/modules/slider.js

```diff
@@ -267,12 +277,22 @@ $.fn.slider = function(parameters) {
           },
           touchEvents: function() {
             module.verbose('Binding touch events');
-            $module.find('.track, .thumb, .inner').on('touchstart' + eventNamespace, function(event) {
+            var downElements = settings.slideEverywhere ? '.thumb,.inner,.track' : '.thumb';
+            $module.find(downElements).on('touchstart' + eventNamespace, function(event) {
               event.stopImmediatePropagation();
               event.preventDefault();
               module.event.down(event);
             });
-            $module.on('touchstart' + eventNamespace, module.event.down);
+            if(!settings.slideEverywhere) {
+              $module.find('.track, .inner').on('click' + eventNamespace, function(event) {
```

Comment:
Shouldn't the event be "touchstart" instead of "click" inside the touchevents?

Reply:
No, "touchstart" is (somehow) equivalent to "mousedown" for touch events.

However, all modern browsers have a built-in mouse emulation, such that they trigger touch and mouse events when a touch screen is used. See this MDN page for more detail on the triggered events: https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Supporting_both_TouchEvent_and_MouseEvent 

Here, we only want to trigger the `up` function when a user taps the slider's track, i.e. they touch and release but do not drag on the track. This is (more or less – see the other discussion) what the browser checks for to trigger the emulated `click` event.